### PR TITLE
Add a community plugin tiddler documenting Projectify

### PR DESCRIPTION
**Pull request by NicolasPetton**
This pull requests adds a community resource tiddler explaining what Projectify is and what features it provides.

### CODE REVIEW

**Comment by Jermolene** [comment]
Thanks @NicolasPetton and apologies for the delay. Pure documentation updates should target the "tiddlywiki-com" branch which enables us to push them immediately to tiddlywiki.com without waiting for the usual release cycle.

**Comment by kookma** [comment]
@NicolasPetton - Please also https://tiddlywiki.com/#%22Notebook%2C%20a%20clean%2C%20uncluttered%20theme%20for%20TiddlyWiki%22%20by%20Nicolas%20Petton and check if the tiddler title and content are consistent and sufficient. I may suggest a shorter tiddler title like Notebook theme or clean them,...
I appreciate to double check.

**Review by OdinJorna — CHANGES_REQUESTED** [review]

File: editions/tw5.com/tiddlers/community/resources/Projectify by Nicolas Petton.tid [hunk 1]
```diff
@@ -0,0 +1,20 @@
+created: 202101061831
+modified: 202101061831
+tags: Resources
```
[the remark this answers is not in the document]
Comment (NicolasPetton) [reply editions/tw5.com/tiddlers/community/resources/Projectify by Nicolas Petton.tid:3]: Thanks, fixed.

**Comment by NicolasPetton** [comment]
@Jermolene I updated the base branch, thanks.

**Comment by NicolasPetton** [comment]
@kookma I'll open a separate pull request to update the tiddler about Notebook.

**Comment by NicolasPetton** [comment]
@Jermolene ping :smile:

**Comment by Jermolene** [comment]
Thanks @NicolasPetton

**Pull request merged**